content/en/profiler/enabling/ruby.md

```diff
@@ -42,6 +42,9 @@ This utility is available on the software repositories of most Linux distributio
 
 Continuous Profiler is not supported on serverless platforms, such as AWS Lambda.
 
+[Single Step Instrumentation](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/) for Linux host or VM and Docker are not supported.
```

Comment:
```suggestion
[Single Step Instrumentation](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/) is not supported for Linux hosts, VMs, or Docker.
```

Reply:
Tweaked link, I think the doc structure slightly changed since I opened the PR:

```suggestion
[Single Step Instrumentation](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/) is not supported for Linux hosts, VMs, or Docker.
```